pkg/types/tuf/v0.0.1/tuf_manifest_v0_0_1_schema.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/tuf/tuf_manifest_v0_0_1_schema.json",
+    "title": "tuf targets payload v0.0.1 Schema",
+    "description": "Schema for tuf manifests",
+    "type": "object",
+    "properties": {
+        "_type": {
+            "description": "Type identifier",
+            "type": "string",
+            "enum": ["root", "targets", "snapshot", "timestamp"]
+        },
+        "version": {
+            "description": "File version",
+            "type": "integer"
+        },
+        "expires": {
+            "description": "Expiration date",
+            "type": "string",
+            "format": "date-time"
+        },
+        "signed": {
+            "description": "Content of the signed manifest",
+            "type": "object",
+            "properties": {
+                "url": {
+                    "description": "Specifies the location of the archive",
+                    "type": "string",
+                    "format": "uri",
+                    "writeOnly": true
+                },
+                "content": {
+                    "description": "Specifies the archive inline within the document",
+                    "type": "string",
+                    "format": "byte",
+                    "writeOnly": true
+                }
+            },
+            "oneOf": [
+                {
+                    "required": [ "url" ]
+                },
+                {
+                    "required": [ "content" ]
+                }
+            ]
+        }
+    },
+    "required": [ "signed" ]
```

Comment:
I might have misunderstood this: the canonical entry should be filling these types in, but the user only needs to provide the content. I think they are required then, since the item in storage should require all three of these pieces.

Reply:
you're correct then - I would mark `_type`, `version`, and `expires` as `readOnly` then since they will be set server side and leave them out of the `required` array.